exercise/static/exercise/results_staff.js

```diff
@@ -1265,6 +1269,10 @@
             // Augment the raw points received from the API by
             // adding colums for modules/_difficulties and filling in missing zeroes
             pointsJson[0].forEach(function(points, index) {
+                // Check Student ID, if it is null then replace it with an em dash
+                if(points['StudentID'] == undefined)
+                    points['StudentID'] = '—';
```

Comment:
Is there a reason to set the mdash value already here? I think it is easily possible to do this in the function `renderParticipantLink`. If the `data` there is null/empty, you could render the mdash. So it is not necessary to prepare the mdash values beforehand on this line.